TVDemo/TVDemo/TVDemo/AppDelegate.swift

```diff
@@ -12,10 +12,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     static let TVBaseURL = "http://localhost:9001/"
     static let TVBootURL = "\(AppDelegate.TVBaseURL)js/application.js"
 
-    func application(_ application: UIApplication,
-                     didFinishLaunchingWithOptions launchOptions: [NSObject : AnyObject]? = [:]) -> Bool {
-
-        window = UIWindow(frame: UIScreen.main().bounds)
+	func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey : Any]? = nil) -> Bool {
```

Comment:
is it correctly indented? if you look to the `static lets` and the `funcs` they are not aligned.